oap-server/server-bootstrap/src/main/resources/fetcher-prom-rules/self.yaml

```diff
@@ -39,42 +39,43 @@ staticConfig:
   labels:
     service: oap-server
 defaultMetricLevel: instance(['service'], ['instance'])
+group: oap
```

Comment:
I didn't find the doc about this `group`. Please recheck.

Reply:
https://github.com/apache/skywalking/pull/5835/files#diff-c6d2ef62107b5e27314760750c667480bcd8b224e16106e32f6bb6b6f3e90144R49-R50